builtin/providers/aws/resource_aws_vpc_dhcp_options.go

```diff
@@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
 	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
 	stateConf := &resource.StateChangeConf{
 		Pending: []string{"pending"},
-		Target:  "",
+		Target:  []string{},
```

Comment:
> The code changed checks that length of target is greater than 0

The issue here seems to be that we're specifying `Target` to be an empty slice, and then this code interprets that as waiting for nothing: 

```
			// If we're waiting for the absence of a thing, then return
			if result == nil && len(conf.Target) == 0 {
				return
			}
```

in this case though, we return `""` (not `nil`). No worries though, I'm cleaning it up and it's only in ~5 places I think